node/src/blockchain/blockchain_interface/blockchain_interface_web3/lower_level_interface_web3.rs

```diff
@@ -431,164 +356,11 @@ mod tests {
         )
     }
 
-    #[test]
-    fn transaction_receipt_works() {
-        let port = find_free_port();
-        let tx_hash =
-            H256::from_str("a128f9ca1e705cc20a936a24a7fa1df73bad6e0aaf58e8e6ffcc154a7cff6e0e")
-                .unwrap();
-        let block_hash =
-            H256::from_str("6d0abccae617442c26104c2bc63d1bc05e1e002e555aec4ab62a46e826b18f18")
-                .unwrap();
-        let block_number = U64::from_str("b0328d").unwrap();
-        let cumulative_gas_used = U256::from_str("60ef").unwrap();
-        let gas_used = U256::from_str("60ef").unwrap();
-        let status = U64::from(0);
-        let tx_receipt_response = ReceiptResponseBuilder::default()
-            .transaction_hash(tx_hash)
-            .block_hash(block_hash)
-            .block_number(block_number)
-            .cumulative_gas_used(cumulative_gas_used)
-            .gas_used(gas_used)
-            .status(status)
-            .build();
-        let _blockchain_client_server = MBCSBuilder::new(port)
-            .raw_response(tx_receipt_response)
-            .start();
-        let subject = make_blockchain_interface_web3(Some(port));
-
-        let result = subject
-            .lower_interface()
-            .get_transaction_receipt(tx_hash)
-            .wait();
-
-        let expected_receipt = TransactionReceipt {
-            transaction_hash: tx_hash,
-            transaction_index: Default::default(),
-            block_hash: Some(block_hash),
-            block_number: Some(block_number),
-            cumulative_gas_used,
-            gas_used: Some(gas_used),
-            contract_address: None,
-            logs: vec![],
-            status: Some(status),
-            root: None,
-            logs_bloom: Default::default(),
-        };
-        assert_eq!(result, Ok(Some(expected_receipt)));
-    }
-
-    #[test]
-    fn get_transaction_receipt_handles_errors() {
-        let port = find_free_port();
-        let subject = make_blockchain_interface_web3(Some(port));
-        let tx_hash = make_tx_hash(4564546);
-
-        let actual_error = subject
-            .lower_interface()
-            .get_transaction_receipt(tx_hash)
-            .wait()
-            .unwrap_err();
-        let error_message = if let BlockchainError::QueryFailed(em) = actual_error {
-            em
-        } else {
-            panic!("Expected BlockchainError::QueryFailed(msg)");
-        };
-        assert_string_contains(
-            error_message.as_str(),
-            "Transport error: Error(Connect, Os { code: ",
-        );
-        assert_string_contains(
-            error_message.as_str(),
-            ", kind: ConnectionRefused, message: ",
-        );
-    }
-
-    #[test]
-    fn transaction_receipt_batch_works() {
```

Comment:
Ok. Could you please add a one-liner to this place (which used to hold the deleted test)? But only if there have remained some similar tests, aimed at failures possibly, which you in turn didn't delete? It's because if anybody find some "sad path" tests they would expect to see them together with a "happy path" test nearby. You deleted a test saying "....something..._works". The word "works" is usually used if the test writer intended to create a full set of tests. It automatically raises eyebrows as it nearly indicates a missing test.  

Just ensure that this impression couldn't easily come on you. If you agree with my reason, use a comment to explain that there is another test somewhere else (the one you refer about in our conversation) which covers it properly.